test/jdk/java/net/NetworkInterface/Test.java

```diff
@@ -24,8 +24,8 @@
 /* @test
  * @bug 4405354 6594296 8058216
  * @library /test/lib
- * @run main Test
- * @run main/othervm -Djava.net.preferIPv4Stack=true Test
+ * @run main Test -Xcheck:jni
+ * @run main/othervm -Djava.net.preferIPv4Stack=true Test -Xcheck:jni
```

Comment:
You will need to put `-Xcheck:jni` before `Test` ( rather than after it ). Also make the first at run tag othervm also, since the test will now run with a command line argument.